app/Http/Controllers/ContactController.php

```diff
@@ -2,9 +2,26 @@
 
 namespace App\Http\Controllers;
 
+use App\Models\Room;
+use App\Models\Contact;
+use App\Models\Reservation;
 use Illuminate\Http\Request;
 
 class ContactController extends Controller
 {
-    //
+    public function viewContactVerify(Request $request){
+        return view('reservation.contact', ['roomId' => $request->roomId]);
+    }
+
+    public function handleVerification(Request $request){
+        $validated = $request->validate([
+            'email' => 'required|email',
+        ]);
+        $room = Room::getRoomData($request->roomId);
+        $contact = Contact::getContact($request->email);
+
+        return ($contact===null) 
```

Comment:
Ik zou de viewReservationCreate route hier returnen, je hoeft in deze functie ook de kamer data niet in te laden, het is logischer als die volgende route dat doet